parameter_files/FLAMINGO.yml

```diff
@@ -0,0 +1,481 @@
+# Values in this section are substituted into the other sections
+Parameters:
+  sim_dir:     /cosma8/data/dp004/flamingo/Runs/
+  output_dir:  /cosma8/data/dp004/FLAMINGO/ScienceRuns/
```

Comment:
I think we should certainly set the input directory since that probably wont change.

I just tried to run this branch on L1000N0900/DMO_FIDUCIAL and already forgot that the output_dir and scratch_dir are just base directories that don't need to include the simulation name. Having an example path in the parameter file might have helped with that.

Reply:
I've added back in some example output directories (under my username, so no-one else will be able to write to them) in https://github.com/SWIFTSIM/SOAP/pull/71/commits/546f95caef21e8fbf7fb3a9ea77069b6c394dd06. I've also put in a check to make sure the output can be written to. This stops someone running all SOAP and then failing to write the output file. I haven't put in a check for the membership files since that's pretty cheap to run anyway.